Specification.md

```diff
@@ -0,0 +1,87 @@
+# XApp Specification
+
+TBD: I suppose it would be nice to call the project XApp instead of xapp ? (used lower case for now)
+
+In order to be shown on the xapp project, applications need to fulfill a list of requirements. This specification is used in order to summarize these requirements and as such can be utilized to decide if an application can join the xapp project.
+
+## Requirements
+
+### Licensing
+
+Xapps need to be licensed on an [OSI-approved license](https://opensource.org/licenses).
+
+### Operating System
+
+Xapps need to be able to run in a Linux environment. The support of other operating systems is optional.
+
+### Graphical Desktop Application
+
+This project targets graphical applications which run on desktop systems and dependencies of them. E.g. console applications, kernel-modules and system-services cannot be xapps.
+
+### Usage of standards
+
+In order to be desktop-environment agnostic, the compliance to the following [freedesktop.org](https://www.freedesktop.org) standards is mandatory to be listed as an xapp:
```

Comment:
I think this requires a little more explanation; to me it reads like all xapps need to comply with these standards, even when they might not need to use them.  Like a calculator app probably won't care about the file manager dbus interface or shared mime database.  Maybe something more like:

"In order to be desktop-environment agnostic, xapps must comply with the following standards when they apply to functionality included in the application."

Reply:
Agreed, thats a better wording .. fixed in a4c89db39fb2e19979d3818defecb26983ce3b87.